modules/src/wiki.py

```diff
@@ -4,9 +4,10 @@ def process(input, entities):
     output = {}
     try:
         query = entities['wikipedia_search_query'][0]['value']
+        page = wikipedia.page(query)
         data = wikipedia.summary(query, sentences=1)
```

Comment:
We're making two queries here!
Read the docs [here](https://wikipedia.readthedocs.org/en/latest/code.html#module-wikipedia) to figure out how to do the same using only one query. (Hint: see the return type of `wikipedia.page`)